site/_docs/history.md

```diff
@@ -65,18 +92,82 @@ other software versions as specified in gradle.properties.
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5478">CALCITE-5478</a>]
+  Use highest input precision for datetimes in `SqlTypeFactoryImpl.leastRestrictive`
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5522">CALCITE-5522</a>]
+  Babel parser cannot handle some overloads of the `DATE_TRUNC` function
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5531">CALCITE-5531</a>]
+  `COALESCE` function throws `ClassCastException`
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5507">CALCITE-5507</a>]
+  `HAVING` alias fails for mixed usage of alias and aggregate function
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5503">CALCITE-5503</a>]
+  `CheapestPlanReplacer` should reuse repeated nodes in a DAG plan
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5468">CALCITE-5468</a>]
+  `SqlToRelConverter` throws if `ORDER BY` contains `IN`
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5515">CALCITE-5515</a>]
+  Add keyspace parameter to `CassandraSchema` and `CassandraTable`
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5416">CALCITE-5416</a>]
+  JDBC adapter for MySQL 5 incorrectly combines `GROUP BY ROLLUP` and `ORDER BY` clauses
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5505">CALCITE-5505</a>]
+  JavaCC warns about missing LOOKAHEAD directives in Parser.jj
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5514">CALCITE-5514</a>]
+  In `RelJson`, add a public `toRex()` instance method
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5442">CALCITE-5442</a>]
+  Tweak janino code generation in `EnumerableInterpretable` to allow debugging
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5521">CALCITE-5521</a>]
+  Remove redundant rowtype check in `RelSubset#add`
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5483">CALCITE-5483</a>]
+  `ProjectAggregateMergeRule` throws exception if literal is non-numeric
+
 #### Build and test suite
 {: #build-1-34-0}
 
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5546">CALCITE-5546</a>]
+  Code style: Break long assignment expressions after '='
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5501">CALCITE-5501</a>]
+  `SqlToRelConverterTest.checkActualAndReferenceFiles` fails intermittently in Jenkins CI
+* Add test for [<a href="https://issues.apache.org/jira/browse/CALCITE-5524">CALCITE-5524</a>] JDBC adapter generates `LIMIT`, `OFFSET` in wrong order for Presto dialect
+* Add tests for [<a href="https://issues.apache.org/jira/browse/CALCITE-2980">CALCITE-2980</a>] Implement the `FORMAT` clause of the `CAST` operator
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5537">CALCITE-5537</a>]
+  Slow test case failures in `LatticeSuggesterTest`
+* Autostyle: Disallow space or newline before ')' in method declaration or call
+* TryThreadLocal values are now not-null by default
```

Comment:
TryThreadLocal is not a build/test issue; should be under the 'minor enhancements' section; also, add back-ticks

Reply:
Fixed in https://github.com/apache/calcite/pull/3103/commits/5c30df57f942b33e8675b6e543202f776a229ae8